apps/src/templates/projects/ProjectCard.story.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import ProjectCard from './ProjectCard';
+
+const PROJECTCARD_NOT_PUBLISHED = {
+    channel: 'ABCDEFGHIJKLM01234',
+    projectName: 'Puppy Playdate',
+    studentName: 'Penelope',
+    studentAge: 8,
+    type: 'applab',
+    updatedAt: '2016-12-31T23:59:59.999-08:00',
+    publishedToPublic: false,
+    publishedToClass: false
+  };
+
+  const PROJECTCARD_CLASS_PUBLISHED = {
+    channel: 'ABCDEFGHIJKLM01234',
+    projectName: 'Puppy Playdate',
+    studentName: 'Penelope',
+    studentAge: 8,
+    type: 'applab',
+    updatedAt: '2016-12-31T23:59:59.999-08:00',
+    publishedToPublic: false,
+    publishedToClass: true
+  };
+
+  const PROJECTCARD_PUBLIC_PUBLISHED = {
+    channel: 'ABCDEFGHIJKLM01234',
+    projectName: 'Puppy Playdate',
+    studentName: 'Penelope',
+    studentAge: 8,
+    type: 'applab',
+    updatedAt: '2016-12-31T23:59:59.999-08:00',
+    publishedToPublic: true,
+    publishedToClass: false
+  };
+
+  const PROJECTCARD_BOTH_PUBLISHED = {
+    channel: 'ABCDEFGHIJKLM01234',
+    projectName: 'Puppy Playdate',
+    studentName: 'Penelope',
+    studentAge: 8,
+    type: 'applab',
+    updatedAt: '2016-12-31T23:59:59.999-08:00',
+    publishedToPublic: true,
+    publishedToClass: true
+  };
```

Comment:
When I have a bunch of nearly identical test data like this, I like to try to make it more explicit what is changing between your test cases. One way you can do that is to come up with a default data, and then just replace the bits that change, so something like this:
```
const defaultData = {
  channel: 'abcdef',
  projectName: 'Puppy Playdate',
  studentName: 'Penelope',
  studentAge: 8,
  type: 'applab',
  updatedAt: 'timestring',
  publishedToPublic: true
  publishedToClass: false
};

const BOTH_PUBLISHED = {
  ...defaultData,
  publishedToClass: true
};
```
This takes advantage of some ES6 syntax of which you may or may not be familiar. It's the equivalent to something like `BOTH_PUBLISHED = Object.assign({}, defaultData, { publishedToClass: true})`.

Both essentially say take the contents of defaultData, then replace publishedToClass with my new value.